datafusion/execution/src/memory_pool/pool.rs

```diff
@@ -247,7 +252,7 @@ mod tests {
         r1.grow(2000);
         assert_eq!(pool.reserved(), 2000);
 
-        let mut r2 = MemoryConsumer::new("s1")
```

Comment:
I renamed the names to match the local variable names as the indirection of a different number was very confusing to me when debugging this test.